packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
+    "system-test": "nyc --exclude=\"error-message.js\" $(npm bin)/mocha ./test/system-test/*.js",
```

Comment:
updated, thanks for catching this. Forgot to merge some other dependent refactoring. The integration tests require a number of env variables to be set so exit behaviour given these variables are not present should work better now.